### PR TITLE
undo reject comment on stream

### PR DESCRIPTION
**Pull request by devonjs**
## What does this PR do?
Allows undo of reject comments in stream

## How do I test this PR?
-Go to stream as moderator
-Click a comment and reject
-Click undo link to bring it back

### CODE REVIEW

**Comment by devonjs** [comment]
@kgardnr update for undo in-stream from #1117

**Comment by kgardnr** [comment]
Thanks so much @devonJS! This is great! We'll merge it right after someone can code review it

**Review by kgardnr — DISMISSED** [review]

File: client/coral-embed-stream/src/components/CommentTombstone.js [hunk 1]
```diff
@@ -32,6 +35,7 @@ class CommentTombstone extends React.Component {
 
 CommentTombstone.propTypes = {
   action: PropTypes.string,
+  onUndo: PropTypes.func,
```
Comment (devonjs) [inline client/coral-embed-stream/src/components/CommentTombstone.js:38]: @kgardnr rebased and refreshed PR to add this line

**Review by alinhle — APPROVED** [review]
Thanks a lot @devonJS. Apologies for the delay in reviewing your work.

**Pull request merged**